java/backup-restore/src/main/java/com/instaclustr/cassandra/backup/aws/S3Backuper.java

```diff
@@ -169,7 +169,7 @@ public void cleanup() {
         try {
             // TODO cleanupMultipartUploads gets access denied, INS-2326 is meant to fix this
             cleanupMultipartUploads();
-
+            transferManager.shutdownNow(true);
```

Comment:
I think this call should be moved outside this try block, or into its own try block if exceptions should be handled.

Right now, if `cleanupMultipartUploads()` fails, which it can due to permissions problems, then the transfer manager won't be shut down cleanly.